LoginPage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Formik, Field, Form } from 'formik';
+import { useHistory } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const LoginPage = () => {
+  const history = useHistory();
+
+  return (
+    <div className="container-fluid">
+      <div className="row">
+        <div className="col-md-6">
+          <img src="image.jpg" alt="Learning" style={{ height: '100vh', width: '100%' }} />
+        </div>
+        <div className="col-md-6">
+          <h2>Login</h2>
+          <p>Enter your email and password to access your account</p>
```

Comment:
The login logic should not be embedded within the component. Consider moving it to a separate function or a service. This would make it easier to test and maintain.